### PR TITLE
Add `clear` and `copy` methods to MutableSequence in Python 3

### PR DESCRIPTION
**Pull request by sproshev**

### CODE REVIEW

File: stdlib/3/typing.pyi [hunk 1]
```diff
@@ -260,6 +260,8 @@ class MutableSequence(Sequence[_T], Generic[_T]):
     def __delitem__(self, i: slice) -> None: ...
     # Mixin methods
     def append(self, object: _T) -> None: ...
+    def clear(self) -> None: ...
```
Comment (JelleZijlstra) [inline stdlib/3/typing.pyi]: I see `clear` in https://github.com/python/cpython/blob/master/Lib/_collections_abc.py, but not `copy`. Why should `copy` be here?

Comment (sproshev) [reply stdlib/3/typing.pyi]: @JelleZijlstra I've decided to put `copy` after reading this https://docs.python.org/3/library/stdtypes.html#mutable-sequence-types

Comment (JelleZijlstra) [reply stdlib/3/typing.pyi]: I'd be inclined to follow the implementation rather than the documentation here.

Comment (sproshev) [reply stdlib/3/typing.pyi]: @JelleZijlstra Agree

**Pull request merged**